lib/src/chart/bar_chart/bar_chart.dart

```diff
@@ -202,21 +202,27 @@ class _BarChartState extends AnimatedWidgetBaseState<BarChart> {
         touchResponse.touchInput is FlPanMoveUpdate ||
         touchResponse.touchInput is FlLongPressStart ||
         touchResponse.touchInput is FlLongPressMoveUpdate) {
-      setState(() {
+
+      if (!widget.data.barTouchData.stickyTouchIndicator) {
         if (touchResponse.spot == null) {
           _showingTouchedTooltips.clear();
```

Comment:
It should be wrapped by `setState()` like previous.

Reply:
i tested this and setstate wasn't doing anything here

Reply:
Okay, I will test it again, 
Thanks!